app/src/main/java/com/framgia/wsm/screen/requestoff/RequestOffViewModel.java

```diff
@@ -798,23 +865,23 @@ public OffRequest getRequestOff() {
         int PM = 1;
     }
 
-    @IntDef({
+    @StringDef({
             TypeOfDays.ANNUAL_LEAVE, TypeOfDays.LEAVE_FOR_CHILD_MARRIAGE,
             TypeOfDays.LEAVE_FOR_MARRIAGE, TypeOfDays.FUNERAL_LEAVE,
             TypeOfDays.LEAVE_FOR_CARE_OF_SICK_CHILD, TypeOfDays.PREGNANCY_EXAMINATON,
             TypeOfDays.SICK_LEAVE, TypeOfDays.MISCARRIAGE_LEAVE, TypeOfDays.MATERNTY_LEAVE,
             TypeOfDays.WIFE_LABOR_LEAVE
     })
     @interface TypeOfDays {
-        int ANNUAL_LEAVE = 0;
-        int LEAVE_FOR_CHILD_MARRIAGE = 1;
-        int LEAVE_FOR_MARRIAGE = 2;
-        int FUNERAL_LEAVE = 3;
-        int LEAVE_FOR_CARE_OF_SICK_CHILD = 4;
-        int PREGNANCY_EXAMINATON = 5;
-        int SICK_LEAVE = 6;
-        int MISCARRIAGE_LEAVE = 7;
-        int MATERNTY_LEAVE = 8;
-        int WIFE_LABOR_LEAVE = 9;
+        String ANNUAL_LEAVE = "";
+        String LEAVE_FOR_CHILD_MARRIAGE = "3";
+        String LEAVE_FOR_MARRIAGE = "2";
+        String FUNERAL_LEAVE = "10";
+        String LEAVE_FOR_CARE_OF_SICK_CHILD = "9";
+        String PREGNANCY_EXAMINATON = "22";
+        String SICK_LEAVE = "19";
+        String MISCARRIAGE_LEAVE = "17";
+        String MATERNTY_LEAVE = "6";
+        String WIFE_LABOR_LEAVE = "14";
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.